api/dummy_data.py

```diff
@@ -0,0 +1,20 @@
+import datetime as dt, random
```

Comment:
We might want to get others to comment but I don't think creating dummy data is necessary. This might startup and kick off the pipeline when we don't want that to happen. The fact that we can do a POST and add an event is fine for now.

Reply:
I agree. If you've got CRUD abilities (or at least CR in this case) there shouldn't be a need for randomly generated data.

In another PR we can add a simple web form for us push values onto the queue. At that time we could make a button to randomly put one of these ids in the events queue.